source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -128,6 +139,42 @@ content: |
             const base64DataKeyId = key.toString('base64');
             console.log("DataKeyId [base64]: ", base64DataKeyId);
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           import pymongo
+           from pymongo import MongoClient
+           from pymongo.encryption_options import AutoEncryptionOpts
+           from bson.codec_options import CodecOptions
+           from bson.binary import (STANDARD)  
+           connection_string = "mongodb://localhost:27017"
```

Comment:
This could be easier to read if there was a line break between the imports and logic.

Reply:
Fixed!